x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -1,11 +1,11 @@
 - module: aws
-  period: 300s
+  period: 1m
```

Comment:
Why changing this to `1m`?

Reply:
Illusion :) Take a look at the whole file: https://github.com/elastic/beats/blob/767a920852c4a996eac421bc5ec9ebf77823d88f/x-pack/metricbeat/module/aws/_meta/config.yml